nemo/core/classes/modelPT.py

```diff
@@ -1206,6 +1208,16 @@ def _update_dataset_config(self, dataset_name: str, config: Optional[Union[DictC
             else:
                 raise ValueError("`dataset_name` when updating config must be one of [train, validation, test]")
 
+    # def configure_ddp(self, model: LightningModule, device_ids: List[int]) -> DistributedDataParallel:
```

Comment:
Why is this not a core level fix ?

Reply:
It was failing unit tests.

Reply:
Hmm wasn't it the default in 1.1.3 anyway ? It's odd that unit tests should fail